packages/rs-drive-abci/src/platform_types/withdrawal/unsigned_withdrawal_txs/v0/mod.rs

```diff
@@ -0,0 +1,140 @@
+//! Withdrawal transactions definitions and processing
+use dpp::dashcore::consensus::Encodable;
+use dpp::dashcore::hashes::Hash;
+use dpp::dashcore::transaction::special_transaction::asset_unlock::qualified_asset_unlock::AssetUnlockPayload;
+use dpp::dashcore::{Transaction, Txid, VarInt};
+use std::fmt::Display;
+use tenderdash_abci::proto::types::VoteExtension;
+use tenderdash_abci::proto::{abci::ExtendVoteExtension, types::VoteExtensionType};
+
+type TxIdAndBytes = (Txid, Vec<u8>);
+
+/// Collection of withdrawal transactions processed at some height/round
+#[derive(Debug, Default, Clone)]
+pub struct UnsignedWithdrawalTxs(Vec<Transaction>);
+
+impl UnsignedWithdrawalTxs {
+    pub fn iter(&self) -> std::slice::Iter<Transaction> {
+        self.0.iter()
+    }
+
+    pub fn into_iter(self) -> std::vec::IntoIter<Transaction> {
+        self.0.into_iter()
+    }
+
+    pub fn len(&self) -> usize {
+        self.0.len()
+    }
+
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
+
+    pub fn from_vec(transactions: Vec<Transaction>) -> Self {
+        Self(transactions)
+    }
+
+    pub fn drain(&mut self) -> UnsignedWithdrawalTxs {
+        Self(self.0.drain(..).collect())
+    }
+}
+
+impl<'a> Display for UnsignedWithdrawalTxs {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_fmt(format_args!("txs:["))?;
+        for tx in &self.0 {
+            f.write_fmt(format_args!("{}", tx.txid().to_hex()))?;
```

Comment:
Tx reverts bytes for hex. So if we just encode extension as a hex they will have different (reversed values)

Reply:
Is this still relevant? We are not encoding extension as hex.